python/lsst/analysis/tools/actions/plot/wholeSkyPlot.py

```diff
@@ -189,18 +252,22 @@ def makePlot(
 
             # Setup figure.
             fig, ax = plt.subplots(1, 1, figsize=self.figureSize, dpi=500)
-            ax.set_xlim(self.xLimits)
-            ax.set_ylim(self.yLimits)
+            xlim, ylim = self._getAxesLimits(ras, decs)
+            ax.set_xlim(xlim)
+            ax.set_ylim(ylim)
```

Comment:
It is probably worth keeping the option of people being able to set their own limits. Defaulting to finding them but with an optional override.